project/Game.py

```diff
@@ -20,10 +20,12 @@ def modify_team(self, oldname, name=None, password=None):
         pass
 
     def add_landmark(self, landmark):
-        pass
+        if landmark not in self.game.landmarks:
+            self.game.landmarks.append(landmark)
```

Comment:
need to return something so that the CLI can figure out success or failure
if landmark is in the list of landmarks there should be an error of some sort

Reply:
I too also did add_landmark. Must be duplicate story in the Product backlog 
i guess we will combine our tests.